files/zh-cn/web/javascript/reference/global_objects/date/index.html

```diff
@@ -131,8 +131,100 @@ <h3 id="实例属性">实例属性</h3>
 </dl>
 
 <h3 id="实例方法">实例方法</h3>
-
-<p>{{ page("/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Date/prototype", "Methods") }}</p>
+<dl>
+  <dt>{{jsxref("Date.prototype.getDate()")}}</dt>
+  <dd>根据本地时间，返回一个指定的 {{jsxref("Date")}} 对象为一个月中的哪一日（`1`-`31`）。</dd>
```

Comment:
Please convert `inline code block` in to html syntax:

``` `1`-`31` ``` --> `<code>1</code>-<code>31</code>`

The same in the following lines ( L138 to L170 )

Reply:
@yin1999 converted